composables/collectionActivity/useCollectionActivity.ts

```diff
@@ -15,7 +15,7 @@ export const useCollectionActivity = ({ collectionId }) => {
 
   const queryPrefix = queryPrefixMap[urlPrefix.value] || 'subsquid'
   const variables = computed(() => ({
-    id: collectionId.value,
+    id: collectionId.value || collectionId.collection,
```

Comment:
It does not make sense... What you should change is how it calls `useCollectionActivity`

check https://github.com/kodadot/nft-gallery/pull/9830/files#diff-329c47d069c9f4255f7e36138bebbe999e113c5fb3dbe0270dde55867341a578R74